Resources/templates/responsive/dashboard/project/invests.php

```diff
@@ -149,9 +154,17 @@
           <td><?= $address ?></td>
           <td><?= $extra_info ?></td>
           <td>
-            <?php if(!$this->project->userIsOwner($invest->getUser())&&!$invest->anonymous): ?>
-            <a data-toggle="modal" href="#messageModal" data-user="<?= $invest->getUser()->id ?>" data-name="<?= $invest->getUser()->name ?>" class="send-private" title="<?= $this->text('support-send-private-message') ?>"><span><?= (int)$this->messages[$invest->getUser()->id] ?></span> <i class="icon-1x icon icon-partners"></i></a>
-            <?php endif ?>
+              <?php $canSendMessages = !$this->project->userIsOwner($invest->getUser())
+                  && !($invest->resign && $invest->anonymous); ?>
+              <?php if ($canSendMessages) { ?>
```

Comment:
could we write ifs with the correspondent : and endif forms? i think it's easier to understand if we are closing and if a foreach or whatever

Reply:
We certainly could write the if that way.

It's a preference for me to use bracers instead of endif. In my opinion, the lines in between an IF opening and closing, should be maximum 4-5.

If that would be the case, it wouldn't really mind in what way we write IFs, FORs, etc.

Feel free to change it your self if you prefer it the other way.